### ISSUE TITLE
Putts won't register on GSPro with EXputt

### ISSUE THREAD

**Issue by Timmander** [springbok/MLM2PRO-GSPro-Connector#148]
![Screenshot 2024-12-01 152358](https://github.com/user-attachments/assets/3e32d7d8-4752-46f6-82e4-eb4a808ee30b)

I can't seem to get putts to register on GSPro using Exputt, even though they are registering on Putting View. When I putt with the EXputt, it registers in putting view but not on GSPro. I've tried adjusting the viewing axis parameters to cover the hitting area, but the aiming indicator on Putting View is pointed to the side and can't be adjusted. 

Any help would be greatly appreciated. 
Thank you

**Comment by wonder99** [comment]
It looks like you have selected Webcam putting rather than Exputt?  There's
no putting view when you use exputt, it uses the Windows Camera app
instead.  The connector will watch that Camera screen for new putts.
Please double check your settings.

**Comment by Timmander** [comment]
![Screenshot 2024-12-01 154844](https://github.com/user-attachments/assets/eb82012a-5a22-47d3-a7a9-93c3a4706972)

Thank you for your reply, is that a setting in the connector itself? I've seen in earlier versions a dropdown to select exputt, but can't seem to find the setting on the connector settings.

**Comment by wonder99** [comment]
LOL, that looks like you're not using this connector.  This one would look
like this

[image: image.png]

**Comment by Timmander** [comment]
I can't see the image but I'm guessing it's this one?
![Screenshot 2024-12-01 174421](https://github.com/user-attachments/assets/3397bdf4-8c06-4983-a42d-26ca5a5d0829)
I set everything up using Joe Lagowski's video and even though everything appears to be working. The putts won't register on GSPro. So far, I've only tried it out in practice mode if that makes a difference. 
![Screenshot 2024-12-01 174429](https://github.com/user-attachments/assets/a3a9e9de-7d53-4c9f-a972-ae84039d37c0)
This is the current putting settings
